pilot/pkg/networking/core/v1alpha3/tls.go

```diff
@@ -203,7 +204,8 @@ func buildSidecarOutboundTLSFilterChainOpts(node *model.Proxy, push *model.PushC
 		out = append(out, &filterChainOpts{
 			sniHosts:         sniHosts,
 			destinationCIDRs: []string{destinationCIDR},
-			networkFilters:   buildOutboundNetworkFiltersWithSingleDestination(push, node, statPrefix, clusterName, "", listenPort, destinationRule),
+			networkFilters: buildOutboundNetworkFiltersWithSingleDestination(push, node, statPrefix, clusterName, "",
+				listenPort, destinationRule, tunnelingconfig.NilBuilder),
```

Comment:
I also doubt why this is nil

Reply:
this is not passthrough here cc @costinm

Reply:
Please take a look at the [comment](https://github.com/istio/istio/pull/37968/files#r872342593) below.

Reply:
I changed it and now tunneling config is applied also in this case.